dependencies/lib-sdl2/builder/Bug5100.patch

```diff
@@ -0,0 +1,33 @@
+--- ./CMakeLists.txt	2020-03-10 18:36:18.000000000 -0700
```

Comment:
What a shame there is still no new SDL release!

Reply:
True. But we can change easily to the new release when it arrives.